ambry-store/src/test/java/com.github.ambry.store/BlobStoreTest.java

```diff
@@ -1248,6 +1248,12 @@ private BlobStore createBlobStore(ReplicaId replicaId, StoreConfig config,
     return StoreTestUtils.createMockReplicaId(storeId, LOG_CAPACITY, filePath);
   }
 
+  /**
+   * Change threshold levels for dynamic ClusterManager block sealing
```

Comment:
didn't understand this comment - what is "ClusterManger block"? Don't think we have this defined anywhere.

Reply:
changed to "dynamic replica sealing"